src/buildMatchSnapshot.js

```diff
@@ -33,4 +42,19 @@ const buildMatchSnapshot = (utils, parseArgs) => {
   };
 };
 
+const safeRequireJestExpect = () => {
+  try {
+    return require("jest-matchers");
```

Comment:
Jest is [planning to rename `jest-matchers` to `expect`](https://github.com/facebook/jest/issues/1679#issuecomment-282478002). Could we return the global `expect` here instead?
```js
if (typeof expect !== "undefined") {
  return expect;
} else {
  return null;
}
```